apps/prairielearn/src/pages/instructorCourseAdminTags/instructorCourseAdminTags.html.tsx

```diff
@@ -1,19 +1,22 @@
-import { html } from '@prairielearn/html';
-import { renderHtml } from '@prairielearn/preact';
+import { z } from 'zod';
+
+import { Hydrate } from '@prairielearn/preact/server';
 
 import { PageLayout } from '../../components/PageLayout.js';
 import { CourseSyncErrorsAndWarnings } from '../../components/SyncErrorsAndWarnings.js';
-import { TagBadge } from '../../components/TagBadge.js';
-import { TagDescription } from '../../components/TagDescription.js';
+import { StaffTagSchema } from '../../lib/client/safe-db-types.js';
 import { type Tag } from '../../lib/db-types.js';
 
+import { InstructorCourseAdminTagsTable } from './components/InstructorCourseAdminTagsTable.js';
+
 export function InstructorCourseAdminTags({
   resLocals,
   tags,
 }: {
   resLocals: Record<string, any>;
   tags: Tag[];
 }) {
+  const StaffTags = z.array(StaffTagSchema).parse(tags);
```

Comment:
`PascalCase` things like this are reserved for Zod schemas or Preact/html components. We could either name this `staffTags` or move the `z....parse(...)` stuff into the prop value below to avoid having to name this at all.